app/vmctl/utils.go

```diff
@@ -5,12 +5,24 @@ import (
 	"fmt"
 	"os"
 	"strings"
+	"time"
 
 	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl/vm"
 )
 
 const barTpl = `{{ blue "%s:" }} {{ counters . }} {{ bar . "[" "█" (cycle . "█") "▒" "]" }} {{ percent . }}`
 
+type timeRange struct {
```

Comment:
I think we can move this part of code and whole code which implements stepper interface to `stepper` folder

Reply:
Refactored to address your suggestions